globals.go

```diff
@@ -63,31 +71,38 @@ const (
 	TaskStarted = "started"
 
 	// The task statuses below indicate that a task has finished.
+	// TaskSucceeded indicates that the task has finished and is successful.
```

Comment:
Can we move the "undispatched", "dispatched", "started", "success", and "failed" statuses in a block at the top of this section in L44 since they're the only actual DB statuses?

Reply:
Done.